### PR TITLE
docs: New config value FORUM_MONGODB_DATABASE added in docs

### PR DESCRIPTION
**Pull request by ChrisChV**
- New Forum config section added in `Configuration and customisation`
- New config value `FORUM_MONGODB_DATABASE` added in docs referencing to https://github.com/overhangio/tutor/pull/453

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -171,6 +171,11 @@ SMTP
 
 Note that the SMTP server shipped with Tutor by default does not implement TLS. With external servers, only one of SSL or TLS should be enabled, at most.
 
+Forum
+*****
+
+- ``FORUM_MONGODB_DATABASE`` (default: ``cs_comments_service``)
```
Comment (regisb) [inline docs/configuration.rst:177]: Can you also please add the ``RUN_FORUM`` and ``FORUM_HOST`` variables?

**Review by regisb — CHANGES_REQUESTED** [review]

File: docs/configuration.rst [hunk 2]
```diff
@@ -171,6 +171,13 @@ SMTP
 
 Note that the SMTP server shipped with Tutor by default does not implement TLS. With external servers, only one of SSL or TLS should be enabled, at most.
 
+Forum
+*****
+
+- ``RUN_FORUM`` (default: ``true``)
+- ``FORUM_HOST`` (default: ``forum``)
+- ``FORUM_MONGODB_DATABASE`` (default: ``cs_comments_service``)
```
Comment (regisb) [inline docs/configuration.rst:179]: Sorry to nitpick @ChrisChV, but could you add "" quotes around the values for `FORUM_HOST` and `FORUM_MONGODB_DATABASE`? This is to indicate that the type of the setting value is a string, as for other items in the docs. Thus:

```
- ``RUN_FORUM`` (default: ``true``)
- ``FORUM_HOST`` (default: ``"forum"``)
- ``FORUM_MONGODB_DATABASE`` (default: ``"cs_comments_service"``)
```

**Comment by ChrisChV** [comment]
Ok @regisb, I think it's done

**Review by regisb — APPROVED** [review]

**Comment by regisb** [comment]
Thanks @ChrisChV!

**Pull request merged**